.github/workflows/tests.yml

```diff
@@ -0,0 +1,111 @@
+---
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        django: [2.2.*, 3.0.*, 3.1.*, 3.2b1]
+        include:
+          - python-version: 3.8
+            django: main
+          - python-version: 3.9
+            django: main
+
+    services:
+      mariadb:
+        image: mariadb:10.1
+        env:
+          MYSQL_ROOT_PASSWORD: mariadb
+        ports:
+          - 3306:3306
+      postgres:
+        image: postgres:10-alpine
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install latest pip, setuptools, and wheel
+        run: |
+          python -m pip install --upgrade pip wheel setuptools
+      - name: Install dependencies
+        run: |
+          PACKAGES=('mock==1.0.1' 'pytest' 'pytest-django' 'pytest-cov' 'django-environ' 'setuptools_scm' 'pyupgrade' 'psycopg2-binary==2.8.4' 'mysqlclient==1.4.5')
```

Comment:
mock 1.0.1 is a bit outdated and the pin should be updated/removed in a follow-up PR.  The pin on psycopg2-binary can probably be loosened to `psycopg2-binary~=2.8.6`

Reply:
I've removed the pins and the tests are still green.  ¯\_(ツ)_/¯

Having no pins on test dependencies has one downside: Consider one of them introduces something that breaks our test setup.  Since there's low number of commits per time on `master` we won't notice it.  Then someone opens a PR and the tests fail.  This makes it harder to figure out if the tests were broken on `master` in the first place or if the PR introduced something breaking.

Pinning everything also won't work without problems.  There's a constant overhead of PR from dependabot (or similar).  And maybe one of the test requirements needs to be installed in different versions depending on django's version.

Keeping the test requirements unpinned as much as possible together with scheduled test runs on master (once per week maybe?) might be a good tradeoff.  `pip` has a dependency resolver by now and can take of installing the correct version of any dependency that depends on a specific version of django.  And we should notice failing tests on master early enough.